Added configuration when using separate sensors per tariff

aetjansen opened the pull request:
Additional configuration example and explanation added for DSMR users, or other setups that have separate source sensors to track for different power tariffs.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

aetjansen committed:
Added configuration when using separate sensors per tariff

Additional configuration example and explanation added for DSMR users, or other setups that have separate source sensors to track for different power tariffs.

klaasnicolaas commented on source/_components/utility_meter.markdown:
```diff
@@ -20,6 +20,7 @@ From a user perspective, utility meters operate in cycles (usually monthly) for
 
 Some utility providers have different tariffs according to time/resource availability/etc. The utility meter enables you to define the various tariffs supported by your utility provider and accounts your consumptions in accordance. When tariffs are defined a new entity will show up indicating the current tariff. In order to change the tariff, the user must call a service, usually through an autom ...
 
+Sensors created with this component are persistent, so values are retained across restarts of home assistant. The first cycle for each sensor, will be incomplete; a sensor tracking daily usage, will start to be accurate the next day after the component was activated. A sensor tracking monthly usage, will present accurate data starting the first of the next month after being added to home assistan ...
```
Pherhaps this in a note?

Funny, I am also busy at the moment to make it work well with the DSMR 😄

aetjansen replied:
Done (although it does not show in preview as such?) Can you please check, @klaasnicolaas ?

klaasnicolaas replied:
https://deploy-preview-8439--home-assistant-docs.netlify.com/components/utility_meter/

aetjansen replied:
thanks! Another n00b question: how would I have known that? It is different then what's shown when I hit the "preview" tab in the editor.. anyway, looks good, thanks

klaasnicolaas replied:
You must look at the two checks below `deploy/netlify`, and click on the Details button. Then you get the Home Assistant (home) website but as new compiled version with your changes.

aetjansen replied:
thanks!

dgomes commented on source/_components/utility_meter.markdown:
````diff
@@ -128,3 +129,37 @@ automation:
+    cycle: monthly
+```
+
+6 different sensors will be created, 2 per sensor; `sensor.daily_energy_peak`, `sensor.daily_energy_offpeak`, `sensor.daily_gas` to track daily consumption and `sensor.monthly_energy_peak`, `sensor.monthly_energy_offpeak`, `sensor.monthly_gas` for monthly cycles.
````
I think it is obvious that 6 sensores will be created since you defined 6 entities.

aetjansen replied:
makes sense, removed the last paragraph.

aetjansen committed:
note created about persistent sensor values

Paragraph about persistent values for this component are now in a note css

aetjansen committed:
Remove info overdose 

No need to explain that 6 new entities are created with the example, as it is clear from the yaml that 6 entities are configured. Removed.

aetjansen commented:
Updated the docs to reflect the feedback.

klaasnicolaas commented:
Thnx for the PR! 👍
